lib/adapters/amqp/subscriber.ts

```diff
@@ -3,7 +3,9 @@ import {ConfigAMQP} from "../../config";
 import {Message} from "../../messageFactory";
 import serviceDetails = require("../../support/serviceDetails");
 
-const _ = require("lodash");
+const clone = require("lodash.clone");
+const isObject = require("lodash.isobject");
+const isString = require("lodash.isstring");
```

Comment:
Hakan suggested to use `typeof s===’string’` instead of `_.isString`. 
Your thoughts about it?

or like this http://stackoverflow.com/a/9436948